app/bundles/CampaignBundle/Command/ExportCampaignCommand.php

```diff
@@ -0,0 +1,287 @@
+<?php
+
```

Comment:
Please declare strict types for all new classes
```suggestion

declare(strict_types=1);

```

Reply:
fixed